bin/emq-jobmanager

```diff
@@ -48,6 +51,8 @@ if __name__ == "__main__":
 
     broker_addr = args.broker_addr
     concurrent_jobs = args.jobs
+    name = args.name
 
-    j = JobManager(queues=queues, concurrent_jobs=concurrent_jobs)
+    j = JobManager(queues=queues, concurrent_jobs=concurrent_jobs,
+                   name=name)
```

Comment:
This will cause the name of the socket to be None if nothing is passed in on the command line.